requirements.txt

```diff
@@ -2,8 +2,12 @@ fastapi==0.115.12
 uvicorn==0.34.0
 kubernetes==32.0.1
 python-dotenv==1.1.0
-pydantic==2.10.6
+pydantic==2.11.2
+pydantic-settings==2.8.1
 pytest==8.3.5
 httpx==0.28.1
 pyyaml==6.0.2
-jinja2==3.1.6
\ No newline at end of file
+jinja2==3.1.6
+aiohttp==3.11.16
+tornado==6.4.2
```

Comment:
What do you need tornado for? Aren't you using fastapi?

Reply:
👍 removed.